src/DependencyInjection/Configuration.php

```diff
@@ -739,6 +740,12 @@ private function createCachePluginNode()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('cache_listeners')
+                    ->info('A list of service ids to act on the response based on the results of the cache check. Must implement '.CacheListener::class.'. Defaults to an empty array.')
+                    ->beforeNormalization()->ifNull()->thenEmptyArray()->castToArray()->ifEmpty()->thenUnset()->end()
```

Comment:
i would rather not unset it but have an empty array as default. that would make the extension class easier.

Reply:
done.

Reply:
did you forget to push? there are 2 entries for cache_listeners in the configuration, and both do `ifEmpty()->thenUnset()`.

Reply:
fixed